src/Model/StoreEmulation.php

```diff
@@ -5,22 +5,32 @@
 namespace Omikron\Factfinder\Model;
 
 use Magento\Framework\App\Area;
+use Magento\Framework\App\AreaList;
 use Magento\Store\Model\App\Emulation;
 
 class StoreEmulation
 {
     /** @var Emulation */
     private $emulation;
+    /**
+     * @var AreaList
+     */
+    private $areaList;
 
-    public function __construct(Emulation $emulation)
+    public function __construct(Emulation $emulation, AreaList $areaList)
     {
         $this->emulation = $emulation;
+        $this->areaList = $areaList;
     }
 
     public function runInStore(int $storeId, callable $proceed)
     {
         try {
             $this->emulation->startEnvironmentEmulation($storeId, Area::AREA_FRONTEND, true);
+
+            $area = $this->areaList->getArea(\Magento\Framework\App\Area::AREA_FRONTEND);
```

Comment:
Could you please use fully-qualified class names here?

Reply:
Sure. Fixed.